lib/sqlalchemy/orm/query.py

```diff
@@ -920,12 +920,7 @@ def get(self, ident):
 
         :param ident: A scalar or tuple value representing
          the primary key.   For a composite primary key,
-         the order of identifiers corresponds in most cases
```

Comment:
should the doc string indicate that the ordered list is still supported (ie keep the old text and just append with the comment about a dict)?

Reply:
yes definitely.